src/pybind/mgr/orchestrator.py

```diff
@@ -835,11 +860,15 @@ def _orchestrator_wait(self, completions):
 
         Waits for writes to be *persistent* but not *effective*.
         """
+        for c in completions:
+            self._update_completion_progress(c, 0.5)
```

Comment:
Why the percentage of completion is "hard-coded"? I think that this value should be provided as one of the completion objects attribute., and each completion be responsible of updating this value each time is checked his "status".

Reply:
one problem I faces was that there is no general solution to accomplish that in general for all orchestrators without modifying every orchestrator individually.

Reply:
Anyway. I can have a look of how to improve this.

Reply:
I have to say I share @jmolmo's concern here. If the progress is always updated by a fixed amount here, it's never going to accurately reflect the actual progress of any given operation